opensrp-pnc/src/main/java/org/smartregister/pnc/repository/PncStillBornRepository.java

```diff
@@ -0,0 +1,62 @@
+package org.smartregister.pnc.repository;
+
+import android.content.ContentValues;
+
+import androidx.annotation.NonNull;
+
+import net.sqlcipher.database.SQLiteDatabase;
+
+import org.apache.commons.lang3.NotImplementedException;
+import org.smartregister.pnc.dao.PncGenericDao;
+import org.smartregister.pnc.pojo.PncStillBorn;
+import org.smartregister.pnc.utils.PncDbConstants;
+import org.smartregister.repository.BaseRepository;
+
+import java.util.List;
+
+public class PncStillBornRepository extends BaseRepository implements PncGenericDao<PncStillBorn> {
```

Comment:
This is not needed, in maternity its combined to one table i.e Maternity Births (Maternity live births + Maternity still born)

Reply:
deleted.